### DISCUSSION TITLE
Can I use panda's `utilities` layer with tailwindcss's `utilities` ?

### DISCUSSION THREAD

**Discussion by minsoo-web** [chakra-ui/panda#1117] [Q&A]
Hi, I'm making a design-system library with `pandacss`.

During the test to confirm that the `tailwindcss` side also works without any problems, 
I found that pandcss's layer name is same with tailwind's layer, name as `utilities`

In index.css file is like below

```css
@tailwind base;
@tailwind components;
@tailwind utilities;

/* It was confirmed that the order had no effect. */
@import "../../../packages/components/dist/styles.css";
```

I think the first solution I thought of would be if the name doesn't overlap with the layer provided by tailwind, is there a way to customize the name of the util layer in pandacss?

Or, is there any other solution you can suggest?

The environment setting of the app code using the package I built with pandacss is as follows. 

vite + typescript + tailwindcss + postcss

**Comment by astahmer** [comment]
I might be wrong, but afaik tailwind doesn't use @layers ? isn't it a custom thing ? so I'm not sure there is going to be a collision, you can probably just try it and see for yourself to make sure you can

**Reply by minsoo-web** [reply]
Thank you for reply 🙇‍♂️
I will make clear example app using codesandbox.

but, before this example making, I will explain what this situation it is.
Here is tailwindcss docs with vite

https://tailwindcss.com/docs/guides/vite

<img width="1114" alt="image" src="https://github.com/chakra-ui/panda/assets/57122180/0c9e4557-5284-498b-a57a-0573a4d1ea25">

As the screenshot above, tailwindcss using `@tailwind`

My app code is below

<img width="922" alt="image" src="https://github.com/chakra-ui/panda/assets/57122180/4afce2da-847a-48da-bf83-a712943a245c">

but, rendered app doesn't has layer `base` and `utilities` it is use case of tailwind's name

<img width="704" alt="image" src="https://github.com/chakra-ui/panda/assets/57122180/40420e7d-6ddb-435b-81ca-899e4d2a3422">

I will make clear re-producing condition like `codesandbox`.

Thank you for your reply again.

**Comment by anubra266** [comment]
https://github.com/chakra-ui/panda/pull/1161
